groups/groups.yaml

```diff
@@ -576,6 +576,7 @@ groups:
     members:
       - avesh.ncsu@gmail.com
       - davanum@gmail.com
+      - mdame@redhat.com
```

Comment:
@damemi please confirm that this is the Google account you want to use for pushing container images to the gcr.io for the descheduler.